packages/vitest/src/node/pkg.ts

```diff
@@ -1,13 +1,16 @@
+import url from 'node:url'
 import c from 'picocolors'
 import { isPackageExists } from 'local-pkg'
 import { EXIT_CODE_RESTART } from '../constants'
 import { isCI } from '../utils/env'
 
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
+
 export async function ensurePackageInstalled(
   dependency: string,
   root: string,
 ) {
-  if (isPackageExists(dependency, { paths: [root] }))
+  if (isPackageExists(dependency, { paths: [root, __dirname] }))
```

Comment:
This change means _all_ calls to `ensurePackageInstalled` will also allow packages that are only dependencies of `vitest`, not whoever has installed `vitest`. Is that good?

An alternative could be to allow the `root` param to be `string | string[]`.

Reply:
This call is needed to check if importing the module will fail. If it's still possible to import (and I assume it is), then it should be fine.